src/windows_sdk.zig

```diff
@@ -576,6 +575,109 @@ pub const ZigWindowsSDK = struct {
 };
 
 const MsvcLibDir = struct {
+    // https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.setup.configuration
+    fn findViaCOM(allocator: std.mem.Allocator) error{ OutOfMemory, PathNotFound }![]const u8 {
+        switch (windows.ole32.CoInitialize(null)) {
+            windows.S_OK, windows.S_FALSE => {},
+            windows.E_OUTOFMEMORY => return error.OutOfMemory,
+            else => return error.PathNotFound,
+        }
+        // TODO: Could calling this be a problem?
+        defer windows.ole32.CoUninitialize();
```

Comment:
I'm unsure what the correct thing to do here is. Note that the C++ version used `CoInitializeEx(NULL, COINIT_MULTITHREADED)` with no `CoUninitialize` call.